docs/new_branch_checklist.md

```diff
@@ -0,0 +1,147 @@
+
+3. Prepare development for the new branch
+
+   - [X] Create release labels: `<branch>/yes`, `<branch>/yes?`, `<branch>/no`, `<branch>/backported`, and `<branch>/conflict`.
+     - [X] Update config/labels.yml with the new labels and features.  Note: the new branch label is color #000000, the N-1 branch label color is #555555, the N-M branch label color is #dddddd
+     - [X] `bin/update_labels.rb`
+
+4. Code changes for new and master branches
+
+   - [X] In `manageiq@<branch>`
+     - [X] Update Gemfile to change the ref for manageiq_plugin, manageiq-gems-pending, and amazon_ssa_support
+     - [X] Update Dockerfile to change the ref to `latest-<branch>`
+     - [X] Update docker-assets/README.md to change the ref
+     - [X] Update VERSION file to `<branch>-pre`
+     - [X] Commit and push the changes
+   - [X] In `manageiq@master`
+     - [X] Update lib/vmdb/appliance.rb CODENAME to `<NextBranch>`
+     - [X] Update lib/vmdb/deprecation.rb version to `<n+2branch>`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-documentation@<branch>`
+     - [X] Update _data/site_menu.yml URLs from `latest` to `<branch>`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-documentation@master`
+     - [X] Update _data/site_menu.yml and remove any `prior:` entries if they exist
+     - [X] Commit and push the changes if they exist
+   - [ ] In `manageiq-api@<branch>`
+     - [ ] Update lib/manageiq/api/version.rb and update the version number
+     - [ ] Commit and push the changes
+   - [ ] In `manageiq-api@master`
+     - [ ] Update lib/manageiq/api/version.rb and update the version number
+     - [ ] Commit and push the changes
+   - [X] In `manageiq-appliance-build@<branch>`
+     - [X] Update bin/nightly-build.sh BRANCH to `<branch>`
+     - [X] Update config/ova.json `vsphere_product_version` to `<branch>`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-appliance-build@master`
+     - [X] Update kickstarts/partials/main/repos.ks.erb `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`
+     - [X] Update kickstarts/partials/post/repos.ks.erb `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`, if necessary
+     - [X] Commit and push the changes
+   - [X] In `manageiq-pods@lasker`
+     - [X] Update all files mentioning `latest` (except the operator `pullPolicy`) to `latest-<branch>`
+     - [X] Update all files mentioning `master` to `<branch>`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-pods@master`
+     - [X] Update images/manageiq-base/Dockerfile
+       - `<branch#>-<branch>` to `<nextbranch#>-<nextbranch>`
+       - `manageiq-release-<branch#>` to `manageiq-release-<nextbranch#>`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-rpm_build@<branch>`
+     - [X] Update config/options.yml ref to `<branch>`
+     - [X] Ensure config/options.yml rpm.version is `<branch#>.0.0`
+     - [X] Commit and push the changes
+   - [X] In `manageiq-rpm_build@master`
```

Comment:
```suggestion
    - [X] Build the manageiq-release RPM and upload it to the builds directory
    - [X] Initial build of the RPM repo including the manageiq-release gem and other general dependencies
   - [X] In `manageiq-rpm_build@master`
```

Reply:
You removed line 77 in your suggestion and so the new lines are in the previous section "In `manageiq-rpm_build@<branch>`".  Was that intentional?

Reply:
Also that makes that @master section empty

Reply:
Fixed